features/rni.feature

```diff
@@ -0,0 +1,187 @@
+      | RNI-deelnemer (07.88.10)        | 101                                         |
+      | Omschrijving verdrag (07.88.20) | Belastingverdrag tussen België en Nederland |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000048 |
+
+    Scenario: wel leveren RNI-deelnemer categorie 7 wanneer er geheimhouding is
+      Gegeven de persoon met burgerservicenummer '000000061' heeft de volgende 'inschrijving' gegevens
+      | naam                            | waarde                                      |
+      | indicatie geheim (70.10)        | 7                                           |
+      | RNI-deelnemer (07.88.10)        | 101                                         |
+      | Omschrijving verdrag (07.88.20) | Belastingverdrag tussen België en Nederland |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000061                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 000000061 |
+      | geheimhoudingPersoonsgegevens | true      |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 101                                             |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Inschrijving                                         |
+
+    Scenario: wel leveren RNI-deelnemer categorie 7 wanneer er opschorting bijhouding is
+      Gegeven de persoon met burgerservicenummer '000000073' heeft de volgende 'inschrijving' gegevens
+      | naam                            | waarde                                      |
+      | indicatie geheim (70.10)        | 0                                           |
+      | RNI-deelnemer (07.88.10)        | 101                                         |
+      | Omschrijving verdrag (07.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'opschortingBijhouding' gegevens
+      | naam               | waarde     |
+      | reden.code         | O          |
```

Comment:
In de tabel 'lo3_pl' is er slechts 1 met opschortingBijhouding gerelateerd veld, nl. 'bijhouding_opschort_reden'. Je kunt hier dus niet de code en de omschrijving opvoeren. Ik weet niet wat voor waarde het veld 'bijhouding_opschort_reden' bevat (een code of een omschrijving) maar afhankelijk daarvan moet je een van beide verwijderen en de andere naam aanvullen met '(67.20)'.

Dat betreffende veld moet dan ook nog toegevoegd worden aan 'tableNameMap' in 'step_defs.js':

	['reden.code [67.20]', 'bijhouding_opschort_reden' ],

of

	['reden.omschrijving [67.20]', 'bijhouding_opschort_reden' ],

Reply:
de gegevens van inschrijving en opschorting samengevoegd.

propertyNameMap heeft:
    ['datum opschorting bijhouding (67.10)', 'datum' ],
    ['reden opschorting bijhouding (67.20)', 'reden.code']